service/systemd/service.go

```diff
@@ -55,24 +55,29 @@ func ListCommand() string {
 	return cmds.listAll()
 }
 
+// Type alias for a DBusAPI factory method.
+type DBusFac = func() (DBusAPI, error)
+
 // Service provides visibility into and control over a systemd service.
 type Service struct {
 	common.Service
 
 	ConfName string
 	UnitName string
-	Dirname  string
+	DirName  string
```

Comment:
As tempting as it is to do drive by fixes for names, it does confuse the diff somewhat for the reviewer.

Reply:
Acknowledged. Pedantry got the better of me.